### PR TITLE
Contact Us Controller

### PR DESCRIPTION
**Pull request by angeljruiz**
- [ ] Specs

### CODE REVIEW

File: config/sidekiq.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+:queues:
+  - default
+  - mailers
```
Comment (andrewc910) [inline config/sidekiq.yml:3]: Nope. I know the benefit and I want you to know this is the proper solution. I am not touching the elastic beanstalk scripts. It's okay if an email takes 30mins to get to us. Please put it in the default queue. I really don't want to spend a day debugging sidekiq on the production server again. But like i said, this really is the proper solution.

File: config/environments/development.rb [hunk 2]
```diff
@@ -10,9 +11,22 @@
...
   config.eager_load = false
+  config.active_job.queue_adapter = :sidekiq
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3001 }
+  config.action_mailer.smtp_settings = {
+    address: 'smtp.gmail.com',
+    port: 587,
+    user_name: ENV['email_username'],
```
[the remark this answers is not in the document]
Comment (angeljruiz) [reply config/environments/development.rb:22]: it's using an account I created. The login details are in the `.env` I posted in slack 🙂

**Comment by andrewc910** [comment]
I just merged the bridge-update branch. Can you fast forward? Also can you fix "app/controllers/contact_us_controller.rb". It looks like there was a merge conflict that hasn't been settled. IDK which is the proper one. It looks like it should be the `perform_async` one so the mail delivery goes through sidekiq but i don't want to break anything.

**Comment by andrewc910** [comment]
Closed for #52

**Pull request closed without merging**